cmd/pr.go

```diff
@@ -88,15 +88,14 @@ func PrCommand(repo *core.Repo, gh func(context.Context) core.Gh) *cli.Command {
 				args = newArgs
 			}
 
-			err = nil
-			localPr, createErr := pr.Create(cCtx.Context, args)
-			if args.CheckoutPr {
```

Comment:
I think I remember that the weird construct with err = nil and createErr was there for a reason but I don't remember anymore 😬 .

Reply:
Yeah, it looked to me like it was about getting around shading of the `err` variable, but indeed with this version it seems it's not needed anymore.